projects/media-viewer/src/lib/store/selectors/bookmarks.selectors.ts

```diff
@@ -30,12 +36,33 @@ export const getEditableBookmark = createSelector(
 export const getBookmarkInfo = createSelector(
   fromAnnotations.getDocumentIdSetId,
```

Comment:
I think it's better to use the document selector to get the documentId rather than annotations